### ISSUE TITLE
作者您好，style_encoder结构中的一个小问题

### ISSUE THREAD

**Issue by zhushuqi2333** [rucmlcv/L2M-GAN#9]
作者您好，有幸读了您的论文，受益匪浅，写得很好，我想将其用于我的工作中，但是现在出现了一个小问题，想请教一下

`out = torch.stack(out, dim=1)  # (batch, num_domains, style_dim)
        idx = torch.LongTensor(range(y.size(0))).to(y.device)
        s = out[idx, y]  # (batch, style_dim)`

就是为什么第一个out要设置成 (batch, num_domains, style_dim)的维度呢，第二维有什么具体含义在里面吗，非常期待您的回复，我十分希望能用上您的这个网络结构~
